### PR TITLE
Bryanthowell ts multitenancy page

### PR DESCRIPTION
**Pull request by bryanthowell-ts**
Added a new page that gives an overview of best practices to establish mulit-tenancy in ThoughtSpot

### CODE REVIEW

File: docs/src/asciidocs/multi-tenancy-best-practices.adoc [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+== Multi-tenancy at the user level
+Each group and user has a "sharing visibility" property, which can be set to *SHAREABLE* or *NOT SHAREABLE*.
+
+When a group is *NOT SHAREABLE*, the users within that group cannot share to that group or to other users in that group.
+
+*NOT SHAREABLE* groups provide access to content (or privileges) without breaking user separation. Each user has no concept they belong to the group - they simply have access to the content.
+
+When a user is set to *SHAREABLE*, other users within the same *SHAREABLE* group as that user can share content to them individually.
+
+An administrator account can share to any group, regardless of the shareable property, while regular users can only share to those groups they belong to that are set to *SHAREABLE*, and *SHAREABLE* users within those groups.
```
Comment (ShashiSubramanya) [inline docs/src/asciidocs/multi-tenancy-best-practices.adoc:40]: Does this make sense? Just trying to simplify and reduce the length of the sentence.
 "An administrator account can share objects with any group, regardless of the shareable property.  The non-admin users can only share with those users and groups that are set as *SHAREABLE*. "

File: docs/src/asciidocs/multi-tenancy-best-practices.adoc [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+
+To allow for multi-tenanted self-service, *SHAREABLE* groups can be created for each organization, so that only the users of each organization can share with one another. The choice of whether to make users shareable or not is up to you - if you prefer all sharing to be done at the group level, then you can set all users to *NOT SHAREABLE*.
```
Comment (ShashiSubramanya) [inline docs/src/asciidocs/multi-tenancy-best-practices.adoc:42]: The choice of whether to make users shareable is up to you; if you prefer all sharing to be done at the group level,  you can set all users to *NOT SHAREABLE*.

**Review by ShashiSubramanya — APPROVED** [review]
Looks good

File: docs/src/asciidocs/multi-tenancy-best-practices.adoc (hunk 1 above)
Comment (bryanthowell-ts) [reply docs/src/asciidocs/multi-tenancy-best-practices.adoc:40]: I like making them shorter sentences but it does need to be as precise as possible. I've changed it to "An administrator account can share objects with any group, regardless of the shareable property. The non-admin users can only share to (1) *SHAREABLE* groups they belong to and (2) *SHAREABLE* users within those groups. *SHAREABLE* users' names are never seen by other users unless both users belong to the same *SHAREABLE* group."

File: docs/src/asciidocs/multi-tenancy-best-practices.adoc [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+:page-title: Multi-tenancy best practices
+:page-pageid: multi-tenancy-best-practices
+:page-description: Multi-tenancy is achieved in ThoughtSpot via group configuration
+
+There are many situations where ThoughtSpot must be configured for multi-tenancy. With ThoughtSpot Everywhere, almost every deployment involves an application developer providing content to many distinct organizations. A ThoughtSpot Enterprise deployment may require separation between departments within the larger company. In both cases, the techniques in this article for creating separation of users and content will apply.
+
+== What is multi-tenancy?
+Multi-tenancy describes a single software system serving users from many distinct organizations, who cannot have awareness of one another or access to each other’s content.
+
+Within this section, a "multi-tenanted" system is one where an administrator see everything at once from a single login, while a "single-tenanted" system does have an “all content” view available, even to an administrator account.
```
[the remark this answers is not in the document]
Comment (bryanthowell-ts) [reply docs/src/asciidocs/multi-tenancy-best-practices.adoc:14]: Added an extra sentence or two around the concept of tenant

File: docs/src/asciidocs/multi-tenancy-best-practices.adoc (hunk 2 above)
Comment (bryanthowell-ts) [reply docs/src/asciidocs/multi-tenancy-best-practices.adoc]: Like this one, made the change

**Pull request merged**